templates/run_template.sh

```diff
@@ -0,0 +1,24 @@
+PROJECT="ansible-service-broker"
+BROKER_IMAGE="ansibleplaybookbundle/ansible-service-broker:latest"
+OPENSHIFT_TARGET="https://kubernetes.default"
+OPENSHIFT_USER="admin"
+OPENSHIFT_PASS="admin"
+DOCKERHUB_USER="CHANGEME"
+DOCKERHUB_PASS="CHANGEME"
+DOCKERHUB_ORG="ansibleplaybookbundle"
```

Comment:
Also...as to the Config...the ideal flow will be we remove these parameters and get them from a ConfigMap, the point of this template work was to get us setup to use a template with the code as it is now.  

I assume after move to ConfigMap we will tweak this template to remove some or all of the parameters.